resource/Resource.md

```diff
@@ -0,0 +1,154 @@
+# Resource API Overview
```

Comment:
When reading "resource" there is also a scenario when you want to record operation on certain resource. Like query all traces that created, updated, read properties of, etc of a certain resource. Where resource may be some business entity. Is this an intendent use of an API?

Reply:
Resource is really about identifying the source of signal data and with that would be generally static. I didn't really see individual signals mutating information about their own source. 
Could you give an example?

Reply:
When Azure creates, modifies, etc. VMs it marks all traces performing operations on them with the VM ID. We call it resource ID =)

I can imagine the same thing applicable everywhere where business defines some resources that has a lifetime.

Reply:
So the traces get tagged with the VM they are coming from?
That would essentially be exactly what this proposal aims at – just that machine identifiers are often to coarse and in the case of metrics, basically always.

Reply:
no, with the VMs they operate on. VM here is a customer VM created via call to Azure. Sorry for confusion

Reply:
Oh I see, the Azure API itself here being what's instrumented.

I think in that case the VM ID would just be part of the signal tag itself, i.e. a tag of trace rather than a tag of the resource of the trace.

Reply:
this comment comes from the very short intro of this document. Maybe make it more explicit what is resource.

Reply:
👍